src/reducers/index.js

```diff
@@ -53,4 +54,5 @@ export default combineReducers({
   permission: PermissionReducer,
   vaccine: VaccineReducer,
   newSensor: NewSensorReducer,
+  sensorDetail: SensorDetailReducer,
```

Comment:
As above, might be nice to use `combineReducers()` on `newSensor` and `sensorDetail` and put them under a common `sensor` header?